src/GameFace/Struct/UserProfile.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameFace.Struct
+{
+    public static class Helper
+    {
+        public static List<UserRecords> AddElement(List<UserRecords> list, UserRecords element)
+        {
+            var found = false;
```

Comment:
+1 for being explicit - some people would argue that you do not have to set found to false as it is the default in C# for a bool, but setting it to false is good practice because not everyone might know the default. The default is also not the same in all languages, so a C++ developer might think that an uninitialised bool is just garbage.

There will be no performance or memory impact on setting it, as the compiler would handle it exactly the same.